stable/kibana/README.md

```diff
@@ -59,6 +59,7 @@ Parameter | Description | Default
 `service.internalPort` | internal port for the service | `4180`
 `service.externalIPs` | external IP addresses | None:
 `service.loadBalancerIP` | Load Balancer IP address (to use with service.type LoadBalancer) | None:
+`service.loadBalancerSourceRanges` | Limit load balancer source IPs to list of CIDRs (where available)) | None:
```

Comment:
let's change the last cell in this row of the table to `[]` so that users know it is an array.